File: python/Benchmark.py
import time
import LinesOfCode


def measure(block):
    start = time.clock()
    block()
    return time.clock() - start


def benchmark(block):
    measures = [measure(block) for _ in range(10)]
    return sum(measures) / 10


millis = benchmark(lambda: LinesOfCode.lines_of_code("./LinesOfCode.py"))
print("Time: ", millis, "ms")

File: ruby/benchmark.rb
require './lines_of_code.rb'

def measure(block)
  start = Time.now
  block
  Time.now - start
end

def benchmark(block)
  (1..10).map {
    block
  }.inject() { |sum = 0, current|
    sum + current
  } / 10
end

time = benchmark(lines_of_code('./lines_of_code.rb'))
puts "Time: #{time}ms"